acceptance/framework/consul/consul_cluster.go

```diff
@@ -85,11 +85,11 @@ func NewHelmCluster(
 
 	logger := terratestLogger.New(logger.TestLogger{})
 
-	// Wait up to 15 min for K8s resources to be in a ready state. Increasing
+	// Wait up to 20 min for K8s resources to be in a ready state. Increasing
 	// this from the default of 5 min could help with flakiness in environments
 	// like AKS where volumes take a long time to mount.
 	extraArgs := map[string][]string{
-		"install": {"--timeout", "15m"},
+		"install": {"--timeout", "20m"},
```

Comment:
I'm not sure if this will help for the test you linked since for the non-secure case helm exists immediately and we use the `WaitForAllPodsToBeReady` function to wait

Reply:
@t-eckert here's the timeout for WaitForAllPodsToBeReady: https://github.com/hashicorp/consul-k8s/blob/3ad09f72f9df397e163abcc24b36feffbb35d454/acceptance/framework/k8s/helpers.go#L46

The ingress gateway pod's logs do show it was registered with consul pretty close to when the test cleaned everything up, which could mean that maybe increasing that timeout a little could help

Reply:
That's what I was thinking. It was very close when the test timed out.

Reply:
So I set this back and I changed the retry counter to timeout at 11 minutes instead of 10. I also fixed the comment to reflect the change.